src/components/PlaybookProgress.js

```diff
@@ -781,11 +781,20 @@ class PlaybookProgress extends Component {
         this.updateGlobalPlaybookStatus(playbookName, playId, STATUS.FAILED);
       }
       // if failed update caller page immediately
-      this.props.updatePageStatus(STATUS.FAILED);
+      this.props.updatePageStatus(STATUS.FAILED, error);
     }
   }
 
-  logMessage = (message) => {
+  logMessage = (message, notAppendNewLine) => {
+    // if have not specified not to append new line break
+    // then if the message does not contain a new line
+    // break append new line break
+    if (!notAppendNewLine) {
+      var hasNewLine = /\r|\n/.exec(message);
+      if (!hasNewLine) {
+        message = '\n' + message + '\n';
```

Comment:
I don't think we should have a newline before the message, or else all of our messages will be double-spaced.

Reply:
@GarySmith ok will remove it